### PR TITLE
Pagure Tests Restructuring

### PR DESCRIPTION
**Pull request by sidpremkumar**

### CODE REVIEW

File: test/pagure_tests/test_pagure.py [hunk 1]
```diff
@@ -0,0 +1,411 @@
+import logging
+from unittest import TestCase
+try:
+    # Python 3 >
+    from unittest.mock import patch  # noqa: F401
+except ImportError:
+    from mock import patch  # noqa: F401
```
Comment (pbortlov) [inline test/pagure_tests/test_pagure.py]: sort out the imports

File: test/pagure_tests/test_pagure.py [hunk 2]
```diff
@@ -0,0 +1,409 @@
...
+        """
+        Tests 'request_reviews' function with repos and:
+            * no last comment,
+            * check_request_state returns False
+            * no errors,
+            * no namespace
+        """
+        # Set up mock return values and side effects
+        def mock_datetime_strptime(date, string):
+            import pdb; pdb.set_trace()
```
Comment (amarza-rh) [inline test/pagure_tests/test_pagure.py]: remove debugging line

File: test/pagure_tests/test_pagure.py [hunk 3]
```diff
@@ -0,0 +1,409 @@
...
+            if string is '%Y-%m-%d %H:%M:%S.%f':
```
Comment (amarza-rh) [inline test/pagure_tests/test_pagure.py]: From pylint: `use ==/!= to compare str, bytes, and int literals [F632] `

File: test/pagure_tests/test_pagure.py [hunk 4]
```diff
@@ -0,0 +1,409 @@
...
+                result = MagicMock()
```
Comment (amarza-rh) [inline test/pagure_tests/test_pagure.py]: variable is assigned but never used

**Pull request merged**